src/compute-client/src/controller/instance.rs

```diff
@@ -2406,12 +2480,12 @@ impl<T: ComputeControllerTimestamp> FrontiersIntrospectionState<T> {
     }
 
     /// Update the introspection state with the given new frontiers.
-    fn update(&mut self, read_frontier: AntichainRef<T>, write_frontier: AntichainRef<T>) {
-        if read_frontier != self.read_frontier.borrow() {
-            self.read_frontier = read_frontier.to_owned();
+    fn update(&mut self, read_frontier: &Antichain<T>, write_frontier: &Antichain<T>) {
+        if read_frontier != &self.read_frontier {
+            self.read_frontier.clone_from(read_frontier);
```

Comment:
:heart:

Reply:
I remembered that myself, but I'm pretty sure clippy would have chastised me if I hadn't :D